CODEOWNERS

```diff
@@ -4,32 +4,55 @@
 
 distribution/ddf-common/*	@stustison
 
+distribution/docs/*         @ricklarsen
+
 
 ###############################
 # CATALOG
 ###############################
 
+catalog/async/*   @peterhuffer  @emmberk
+
+catalog/core/catalog-core-api/*             @brendan-hofmann
```

Comment:
Its the actual folder path, so it would be `catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/types/*`